doc/introduction/circuits.rst

```diff
@@ -167,8 +172,58 @@ For example:
     result = circuit(0.543)
 
 
-Quantum circuits from other frameworks
---------------------------------------
+Collections of QNodes
+---------------------
+
+Sometimes you may need multiple QNodes in your variational algorithm. This is
```

Comment:
Say clearer here what the use case is. IMO it is not multiple QNodes in gerenal, but of a similar form. The paragraph also sounds a bit confusing. Maybe try this:

"Sometimes you may need multiple QNodes that only differ in the measurement observable (like in VQE), or in the device they are run on (for example, if you benchmark different devices). To save you from defining every single QNode "by hand", PennyLane offers QNode *collections* as a convenient way to define and run families of QNodes.

QNode collections are sets of QNodes that have the same signature. We introduce this by an example. Consider the following quantum function ansatz:

...

"

Reply:
Good call!